src/routes/blog/+page.svelte

```diff
@@ -180,7 +180,8 @@
                                 {@const author = data.authors.find(
                                     (author) => author.slug === post.author
                                 )}
-                                {#if author}
+                                {@const draft = post.draft}
```

Comment:
why do we need to declare this variable?

Reply:
Do you think the code will look cleaner with or without?

Reply:
I think it's fine without.

Reply:
Updated